summernote-ext-print.js

```diff
@@ -69,7 +69,7 @@
 
         this.$printframe = $(
           '<iframe name="summernotePrintFrame"' +
-          'width="0" height="0" frameborder="0" src="about:blank" style="visibility:hidden">' +
+          'width="0" height="0" frameborder="0" src="about:blank" style="display:none">' +
```

Comment:
Did you rollback the style of this hidden frame intentionally?
I fixed this for https://github.com/lqez/summernote-ext-print/issues/3